src/github/nag.rs

```diff
@@ -143,6 +158,26 @@ fn update_proposal_review_status(proposal_id: i32) -> DashResult<()> {
     Ok(())
 }
 
+fn remove_issue_label(issue: &Issue, label: &str) {
+    let _ = GH.remove_label(&issue.repository, issue.number, label);
+}
+
+fn remove_issue_labels(issue: &Issue, labels: &[&str]) {
+    for label in labels {
+        remove_issue_label(issue, label);
+    }
+}
+
+fn add_issue_label(issue: &Issue, label: &str) -> DashResult<()> {
+    GH.add_label(&issue.repository, issue.number, label)
+}
+
+fn close_pull_request(issue: &Issue) {
```

Comment:
This fn signature makes me sad that PRs are represented as Issues internally. At a minimum can you make a TODO or open an issue to make this type-safe and only apply to PRs but not issues? If `GH.close_pr` will correctly work on an issue even though it hits the `pulls` endpoint, can we call all of this `close_issue` and continue assuming issues and PRs are roughly the same?